intro-HPC/leuven/ch_hpc_policies.tex

```diff
@@ -0,0 +1,77 @@
+\chapter{HPC Policies}
+\label{ch:hpc-policies}
+
+The cluster has been setup to support the ongoing research in all the domains
+at the \university.  As such, it should be considered as
+valuable research equipment.  User shall not abuse the system for other purposes. By
+registering, users accept this implicit user agreement.
+
+In order to shared resources in a fair way, a number of site policies have been
+implemented:
+
+\begin{enumerate}
+\item  Single user node policy
+\item  Priority based scheduling
+\item  Fairshare mechanism
+\item  Crediting system
+\end{enumerate}
+
+
+\section{Single user node policy}
+
+Each user will have exclusive access to the nodes that are assigned to him.
+This means that no jobs from other users will be executed on the nodes during
+your computations.  Of course, it is very likely that multiple jobs from the
```

Comment:
Maybe use `possible` (or `quite possible`), since this depends entirely on the requested and available resources?